### PR TITLE
Add weight_only support for PyTorch framework

### PR DESCRIPTION
**Pull request by PenghuiCheng**
## Type of Change

Add weight_only support for PyTorch framework
No API changed

## Description

Add weight_only support for PyTorch framework

## Expected Behavior & Potential Risk

Support weight_only quantization

### CODE REVIEW

**Review by xin3he — APPROVED** [review]

File: intel_extension_for_transformers/llm/quantization/nn/modules.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+                * self.scaling[self.active_adapter]
+            )
+
+        return out
+
+    def set_weights_bias(self, weight_data, bias=None):
+        self.weight = ParamsQBits(
+            weight_data, requires_grad=False, compress_statistics=self.compress_statistics, quant_dtype=self.weight_dtype
+        )
+        torch.ops.weight_only_jblasop.jblas_symqdq_weight(self.weight, False, 4, 32) # TODO: change to 4bit quantize instead of qdq
```
Comment (zhewang1-intc) [inline intel_extension_for_transformers/llm/quantization/nn/modules.py:163]: have we added the uts for this pr? jblas_symqdq_weight has been replaced with jblas_dequantize now.

Comment (PenghuiCheng) [reply intel_extension_for_transformers/llm/quantization/nn/modules.py]: we will add training part for this function， it will cover this.

**Review by lvliang-intel — APPROVED** [review]

**Comment by VincyZhang** [comment]
coverage drop due to newly added files not add UT for them @PenghuiCheng 
image

**Comment by VincyZhang** [comment]
Will merge for release test first, but need to add UT to increase coverage by next Tuesday. @PenghuiCheng

**Pull request merged**